README.md

```diff
@@ -1,2 +1,24 @@
-# PSPs
-Polkadot Standards Proposals
+# Polkadot Standards Proposals (PSPs)
+
+
+A Polkadot Standards Proposal (PSP) describes standards and improvements for the Polkadot ecosystem. The Polkadot Standards Proposal GitHub is a community-based initiative.  
+
+## PSPs Work Flow
+
+Below is the workflow of a successful PSP:
+```
+1. Draft -> 2. Call for Feedback -> 3. Accepted -> 4. Integrated
+```
+1. **Draft:** A first draft, which is merged into this repo and actively improved together with the community. 
+2. **Call for Feedback:** The PSP will be shared on different channels for additional feedback for at least 2 weeks. The result of this step is either an acceptance of the standard (->Accepted) or the rejection (->Draft). 
+3. **Accepted:** Any further changes are unlikely, and developers can start integrating the PSP. Once a PSP is accepted, a reference implementation needs to be created.    
+4. **Integrated:** The PSP is actively used and a reference implementation exists. . 
+
+## Contributing PSPs
+
+Before you start writing a formal PSP, you should discuss an idea in the various community channels. A PSP should provide the motivation as well as a technical specification for the feature. 
```

Comment:
Perhaps we could add some links to the various community channels here (Riot, forum, etc.)

Reply:
Is this good: https://polkadot.network/community/ ? Or do we have a page on wiki with more technical selection?

Reply:
I added a link to the community website. Should be sufficient for now.